SWLOR.Game.Server/Perk/MartialArts/CircleKick.cs

```diff
@@ -80,8 +80,11 @@ private void ApplyFeatChanges(NWPlayer oPC, NWItem unequippingItem)
         {
             NWItem mainHand = oPC.RightHand;
             NWItem offHand = oPC.LeftHand;
+            NWItem fist = oPC.Arms;
             CustomItemType mainType = mainHand.CustomItemType;
             CustomItemType offType = offHand.CustomItemType;
+            CustomItemType glovesType = fist.CustomItemType;
```

Comment:
I see where you're getting the gloves type but don't seem to be doing anything with it? The original code's check was looking for the absence of a weapon. I'm not sure you need to check the glove in this case since we only care about what's equipped?